server/service/devices.go

```diff
@@ -684,7 +735,9 @@ func fleetdError(ctx context.Context, request interface{}, svc fleet.Service) (f
 }
 
 func (svc *Service) LogFleetdError(ctx context.Context, fleetdError fleet.FleetdError) error {
```

Comment:
Do we need this endpoint exposed to this auth at all? Seems like it isn't called from anywhere in iOS/iPadOS self-service.